tests/apps/flask_app/service-account-info.json

```diff
@@ -0,0 +1,2 @@
+# create a service account for Pub/Sub on Google Cloud Console and replace 
```

Comment:
I'd rather suggest we don't commit this file to avoid leaking secrets by accident

Reply:
LGTM